### DISCUSSION TITLE
Confuse entitäten HmIP-FALMOT-C12

### DISCUSSION THREAD

**Discussion by Moshkopp** [SukramJ/aiohomematic#787] [Q&A] [answered]
Hallo, 

Ich habe heute meinen HmIP-FALMOT-C12 bekommen, die Installation RaspMatic und läuft Problemlos. 
Ich bekomme auch entitäten diese bestehen aber lediglich aus Sensoren. 

![d](https://user-images.githubusercontent.com/24791143/209821500-9e4bb190-4e19-4828-bef3-5f7cc85de073.png)

Ich hatte gehofft das die Stellantriebe als Climate eingebunden werden, damit man diese auch steuern könnte. 
Ist das nicht möglich oder sollte es so wie es bei mir aussieht, garnicht aussehen. 

`{
  "home_assistant": {
    "installation_type": "Home Assistant OS",
    "version": "2022.12.8",
    "dev": false,
    "hassio": true,
    "virtualenv": false,
    "python_version": "3.10.7",
    "docker": true,
    "arch": "x86_64",
    "timezone": "Europe/Berlin",
    "os_name": "Linux",
    "os_version": "5.15.80",
    "supervisor": "2022.12.1",
    "host_os": "Home Assistant OS 9.4",
    "docker_version": "20.10.19",
    "chassis": "embedded",
    "run_as_root": true
  },
  "custom_components": {
    "better_thermostat": {
      "version": "1.0.0-beta55",
      "requirements": []
    },
    "hacs": {
      "version": "1.29.0",
      "requirements": [
        "aiogithubapi>=22.10.1"
      ]
    },
    "homematicip_local": {
      "version": "1.25.1",
      "requirements": [
        "hahomematic==2022.12.8"
      ]
    }`

**Comment by SukramJ** [comment, marked as answer]
Der HmIP-FALMOT-C12 lässt eine Steuerung nur über gekoppelte Wandthermostate zu. 
Eine manuelle Bedienung ist nicht möglich. Aus HA Sicht ist der HmIP-FALMOT-C12 damit nur eine Sammlung von Sensoren.
Am Besten dazu mal im [Homematic Forum](https://homematic-forum.de/) nachschauen.

Um als `climate` Entität in HA darstellbar zu sein benötigt man mindestens IST und SOLL Temperatur.

**Comment by Moshkopp** [comment]
Ok, das heißt es müssen zwingend die von Homematic sein, und die Wandthermostate könnte ich dann mittels HA automatisieren.

**Reply by SukramJ** [reply]
Richtig

**Reply by Moshkopp** [reply]
Vielen Dank für die schnelle antwort.
